CMakeLists.txt

```diff
@@ -90,7 +90,7 @@ if (${MINIMAX_COMPONENT})
    add_subdirectory(GX-TimeFrequency)
 endif ()
 
-option (LBASIS_COMPONENT "Enable the localized basis component" OFF) 
+option (LBASIS_COMPONENT "Enable the localized basis component" ON) 
```

Comment:
better set this OFF as default because otherwise it is built in the cp2k automatic toolchain script

Reply:
set OFF